snap/README.md

```diff
@@ -1,212 +1,38 @@
-# EdgeX Foundry App Service Configurable Snap
-[![snap store badge](https://raw.githubusercontent.com/snapcore/snap-store-badges/master/EN/%5BEN%5D-snap-store-black-uneditable.png)](https://snapcraft.io/edgex-app-service-configurable)
+# EdgeX App Service Configurable Snap
+[![edgex-app-service-configurable](https://snapcraft.io/edgex-app-service-configurable/badge.svg)][edgex-app-service-configurable]
 
-This folder contains snap packaging for the EdgeX Foundry's App Service Configurable application service.
+This directory contains the snap packaging of the EdgeX App Service Configurable application service.
 
-The project maintains a rolling release of the snap on the `edge` channel that is rebuilt and published at least once daily.
+The snap is built automatically and published on the Snap Store as [edgex-app-service-configurable].
 
-The snap currently supports both `amd64` and `arm64` platforms.
+For usage instructions, please refer to [docs].
```

Comment:
For usage instructions including installation, configuration, managing Services, debugging, and so on, please refer to docs.

Reply:
These all loosely fall under "usage". Once the reader goes to docs, they will find the details.